fio.yaml

```
@@ -0,0 +1,41 @@
+package:
+  name: fio
+  version: "3.37"
+  epoch: 0
+  description: Flexible I/O Tester
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - coreutils
+      - libaio-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/axboe/fio
+      tag: fio-${{package.version}}
+      expected-commit: 028fcc7db00155fe329497d7501184e8fe49342f
+
+  - runs: |
+      ./configure --prefix=/usr
+      make
+      make DESTDIR="${{targets.contextdir}}" install
+
+subpackages:
+  - name: fio-docs
+    pipeline:
+      - uses: split/manpages
+
+update:
+  enabled: true
+  github:
+    identifier: axboe/fio
+    strip-prefix: fio-
```

Lets add a basic test 
```suggestion
    strip-prefix: fio-
 
test:
    pipeline:
        - name: Verify fio installation
          runs: |
            fio --version || exit 1   
```